.github/workflows/build-test-deploy.yml

```diff
@@ -592,41 +620,23 @@ jobs:
           command: nix -- build .#acala-node
           token: "${{ secrets.CACHIX_AUTH_TOKEN }}"
 
-  cache-kusama-parachains-devnet:
-    name: "Package kusama parachains devnet"
-    needs:
-      - package-polkadot-node
-    runs-on:
-      - self-hosted
-      - linux
-      - x64
-      - sre
-    concurrency:
-      group: ${{ github.workflow }}-cache-kusama-parachains-devnet-${{ github.ref }}
-      cancel-in-progress: true
-    continue-on-error: true
-    container:
-      image: niteo/nixpkgs-nixos-22.05:316b762afdb9e142a803f29c49a88b4a47db80ee
-    steps:
-      - uses: actions/checkout@v3
-      - uses: "./.github/templates/watch-exec"
-        with:
-          command: nix -- build .#kusama-picasso-karura-devnet
-          token: "${{ secrets.CACHIX_AUTH_TOKEN }}"
-
   cache-devnet-all-dev-local:
-    name: "Package Full Devnet"
+    name: "Cache devnet all dev local"
```

Comment:
```suggestion
    name: "Build all devnets"
```

Reply:
this name seems simpler and better. 
But imho name should be same as display name.  `Build all devnets` = `build-all-devnets`.  
Why would somebody want to have name != display name?

Reply:
changed `Cache devnet all dev local` -> `Build all devnets` and `cache-devnet-all-dev-local` -> `build-all-devnets` in next PR.